### PR TITLE
Add support for Pyuploadcare 4.2.x new features

### PR DESCRIPTION
**Pull request by evgkirov**
## Description

Related [pyuploadcare](https://github.com/uploadcare/pyuploadcare) issues:

- [#239 save_URL_duplicates and check_URL_duplicates are not supported](https://github.com/uploadcare/pyuploadcare/issues/239)
- [#258 multipage conversion (save_in_group param in documentJobSubmitParameters)](https://github.com/uploadcare/pyuploadcare/issues/258)
- [#259 Update webhooks API (event types and versioning)](https://github.com/uploadcare/pyuploadcare/issues/259)
- [#260 Add AWS Rekognition Moderation](https://github.com/uploadcare/pyuploadcare/issues/260)

## Checklist

- [ ] Tests (if applicable)
- [ ] Documentation (if applicable)
- [ ] Changelog stub (or use [conventional commit messages](https://www.conventionalcommits.org/))

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,14 +1,14 @@
 [tool.poetry]
 name = "pyuploadcare-example"
-version = "0.2.0"
+version = "4.2.0"
```
Comment (rsedykh) [inline pyproject.toml:3]: Is it pyuploadcare or pyuploadcare-example version?

**Review by rsedykh — APPROVED** [review]
Just one question.

File: pyproject.toml (hunk 1 above)
Comment (evgkirov) [reply pyproject.toml:3]: pyuploadcare-example

Why don't we synchronize these versions?

Comment (rsedykh) [reply pyproject.toml:3]: It's a good idea, I'm more than ok with it.

**Review by dmitry-mukhin — APPROVED** [review]

**Pull request merged**